recipe/meta.yaml

```diff
@@ -23,14 +23,14 @@ requirements:
     - {{ compiler('cxx') }}
   host:
     - sqlite
-    - rdkit ==2024.03.4
-    - libboost-devel
+    - librdkit-dev ==2024.09.6
+    - libboost-devel  >=1.86.0,<1.87.0a0
```

Comment:
Please don't do this manually, the boost version is handled by the global pinning and the migrator infrastructure. The only thing needed would have been #29. :)

Reply:
Hi @h-vetinari 
Thanks a lot for this comment. I wasn't really fond of my workaround, but I didn't know about the global pinning and I thought the issues I experienced were due to some misconfigured dependency in librdkit-dev + librdkit. I'm happy to learn I was wrong.